src/Menu.js

```diff
@@ -28,12 +28,18 @@ const Menu = ({ children, className, dataTest }) => (
             .menu-list-wrapper {
                 padding: ${spacers.dp4} 0;
             }
+            div {
+                display: inline-block;
+                min-width: 128px;
+                max-width: ${width};
```

Comment:
This line should be in its own `<styled jsx>` tag because of how styled-jsx handles static vs. dynamic CSS[1]:

```
        <style jsx>{`
            .menu-list-wrapper {
                padding: ${spacers.dp4} 0;
            }
            div {
                display: inline-block;
                min-width: 128px;
            }
        `}</style>
        <style jsx>{`
            div {
                max-width: ${width};
            }
        `}</style>
```

[1] https://github.com/zeit/styled-jsx#via-interpolated-dynamic-props

Reply:
Oh, that makes sense. Thanks for the reference.